.eslintrc.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Rules to adopt new design system
+ *
+ * class list will be updated once the
+ * relevant class is completed removed from
+ * tailwind-design-system.ts
+ *
+ */
+const getRules = () => {
```

Comment:
can we move this to a separate file? we might have more custom rules in the future.
having separate plugins for each purpose makes more sense to me.